src/Mimi.jl

```diff
@@ -225,6 +228,7 @@ function addcomponent(m::Model, t, name::Symbol;before=nothing,after=nothing)
     end
 
     ComponentReference(m, name)
+    add_node(m.model_graph, name)
 end
```

Comment:
This needs to be above the previous line. The last line of a function is its return value (unless you specify a return value explicitly with ``return x``), so this line changes the return value of the function (and actually that breaks the units tests.